packages/core/src/browser/shell/tab-bar-toolbar/tab-bar-toolbar.tsx

```diff
@@ -60,13 +64,22 @@ export class TabBarToolbar extends ReactWidget {
     updateItems(items: Array<TabBarToolbarItem | ReactTabBarToolbarItem>, current: Widget | undefined): void {
         this.inline.clear();
         this.more.clear();
+
+        const contextKeys: Set<string> = new Set();
```

Comment:
minor: we can simplify and infer the type

```suggestion
        const contextKeys = new Set<string>();
```